src/styles/card.scss

```diff
@@ -0,0 +1,49 @@
+.cardContainer {
+  width: 100px;
+  height: 100px;
+  margin: 5px;
+  perspective: 600px;
+
+  &.disappear {
+    visibility: hidden;
+  }
+
+  .dummy {
```

Comment:
Is `dummy` being used anywhere? If yes then ignore, if not then remove un-used class.